packages/react-server-test-pages/pages/data/delay.js

```diff
@@ -7,10 +7,9 @@ export default class DelayDataPage {
 	getContentType() {
 		return 'application/json';
 	}
-	handleRoute(next) {
-		return Q.delay(this.getRequest().getQuery().ms||0).then(next);
-	}
 	getResponseData() {
-		return JSON.stringify({'ok': true});
+		const { ms, val } = this.getRequest().getQuery();
+		return Q.delay(ms||0)
+			.then(() => val||JSON.stringify({'ok':true}));
```

Comment:
Added a `val` parameter to `/data/delay`.  It just gets echoed back after `ms` milliseconds.